packages/frontend/package.json

```diff
@@ -53,14 +53,14 @@
         "katex": "^0.16.22",
         "lucide-solid": "^0.471.0",
         "notebook-types": "link:../notebook-types/pkg",
-        "prosemirror-commands": "^1.5.2",
-        "prosemirror-inputrules": "^1.4.0",
-        "prosemirror-keymap": "^1.2.2",
-        "prosemirror-model": "^1.23.0",
+        "prosemirror-commands": "^1.7.1",
+        "prosemirror-inputrules": "^1.5.0",
+        "prosemirror-keymap": "^1.2.3",
+        "prosemirror-model": "^1.25.2",
```

Comment:
Bumped prosemirror-model and -view to agree with automerge-prosemirror's [`package.json`](https://github.com/automerge/automerge-prosemirror/blob/7f8afffc3d764a8d5cf43f9ec47c50af4b427f04/package.json).

This made some errors go away.